### ISSUE TITLE
DNS Blocking Not Working

### ISSUE THREAD

**Issue by wrdixon** [ReliefLabs/EasyTomato#52]
DNS requests weren't making to the FORWARD chain, but were getting sucked out in the INPUT chain.  On the advanced-dhcpdns.asp page we have Use internal DNS and Intercept DNS port (UDP 53) checked.  These make it so that if someone selects their own DNS servers on their computer, we can intercept that and force them to go to OpenDNS or whatever.  Well, so this also means that no longer is the DNS request a Forward, but its actually originating from the router, as the router is physically doing the lookup for the computer.

Basically we just need to put a line in the INPUT that catches port 53 request and sends them down to the rres00 chain.

**Comment by EagleEye905** [comment]
Does this mean that EasyTomato overrides the DNS server that someone sets up on their own computer?  And always send to OpenDNS if the router is configured that way?

Thanks!
